src/impl.h

```diff
@@ -155,37 +155,41 @@ struct Manifold::Impl {
     }
 
     Vec<TriRef> triRef;
-    if (!meshGL.runOriginalID.empty()) {
```

Comment:
So if I understand this correctly, this is mainly just removing the conditional?

Reply:
Yes, I realized this block was basically a better `InitializeOriginal`, so instead of conditionally doing this block or that initialization, I removed the other and am using this one all the time, which fixed a bug.